projects/laji/src/app/+observation/form/observation-form.component.html

```diff
@@ -329,7 +329,7 @@
                 max="5"
                 list="accuracyPowers"
                 (click)="enableAccuracySlider()"
-                (change)="onAccuracySliderChange()"
+                (input)="onAccuracySliderChange()"
```

Comment:
I'd probably change the name here to reflect that the event has changed to `input` i.e. `onAccuracySliderInput()`